.github/workflows/test.yml

```diff
@@ -46,16 +46,16 @@ jobs:
                   yarn install --frozen-lockfile
             - name: Run ESLint
               run: |
-                  yarn netlify-zip-it-ship-it-issue-38-workaround && yarn start && yarn run test:eslint
```

Comment:
Hah, I guess the very verbose name that I chose for this Yarn script ended up coming in handy. Looks like the issue is fixed.